src/services/services.ts

```diff
@@ -4694,6 +4694,10 @@ namespace ts {
                         if (!(container.kind === SyntaxKind.ModuleBlock || container.kind === SyntaxKind.SourceFile)) {
                             return undefined;
                         }
+                    } else if (modifier === SyntaxKind.AbstractKeyword) {
+                        if (!(container.kind === SyntaxKind.ClassDeclaration || declaration.kind === SyntaxKind.ClassDeclaration)) {

```

Comment:
Btw: This is what you meant with `ClassExpression` right?

    function yolo() {
        abstract class test {
            foo: number;
        }
    }

I get an error that `Modifiers cannot appear here`